__tests__/index.test.js

```diff
@@ -199,3 +199,23 @@ describe('/api/articles/:id/comments', () => {
     expect(errText).toBe('Bad Request')
   })
 })
+
+describe('/api/comments/:id', () => {
+  it('DELETE:204 issues a response with no content', async () => {
+    return await request(app).delete('/api/comments/1').expect(204)
+  })
+
+  it('DELETE:400 (invalid ID) issues a response with `Bad Request` error text', async () => {
+    const res = await request(app).delete('/api/comments/a').expect(400)
+
+    const errText = res.error.text
```

Comment:
The `res.error` is part of supertest and although by checking this you are asserting that supertest got the right status code FYI. This is client specific and won't be available to different clients. 

If you want this to be part of your response then you should send a response body with the required text.